packages/mdc-textfield/README.md

```diff
@@ -299,6 +299,7 @@ Method Signature | Description
 `isRequired() => boolean` | Returns whether the input is required.
 `setRequired(isRequired: boolean)` | Sets whether the input is required.
 `handleTextFieldInteraction(evt: Event) => void` | Handles click and keydown events originating from inside the Text Field component.
+`handleValidationAttributeMutation(mutationsList: {Array<MutationRecord>}) => void` | Handles validation attribute changes.
```

Comment:
As per an offline conversation, this type should actually non-nullable:

```js
handleValidationAttributeMutation(mutationsList: !Array<MutationRecord>)
```

Reply:
Done. Thanks!